File: java/src/main/java/challenges/aoc2024/day7/Application.java
package challenges.aoc2024.day7;

import challenges.aoc2024.day7.service.CalibrationService;
import challenges.aoc2024.day7.service.DaySevenSolver;
import service.TaskSolver;

public class Application {
    public static void main(String[] args) {
        final TaskSolver solver = new TaskSolver(new DaySevenSolver(new CalibrationService()));
        solver.solveTasks();
    }
}

File: scala/src/main/scala/challenges/aoc2025/day4/Application.scala
package challenges.aoc2025.day4

import _root_.service.TaskSolver
import challenges.aoc2025.day4.service.{DayFourSolver, ForkliftService}

object Application {
  def main(args: Array[String]): Unit = {
    val solver: TaskSolver = TaskSolver(DayFourSolver(ForkliftService()))
    solver.solveTasks()
  }
}
